optaplanner-core/src/main/java/org/optaplanner/core/impl/score/buildin/bendable/BendableScoreDefinition.java

```diff
@@ -44,6 +44,11 @@ public BendableScore getZeroScore() {
         return BendableScore.zero(hardLevelsSize, softLevelsSize);
     }
 
+    @Override
+    protected Number getOne() {
+        return 1;
```

Comment:
Why BendableScoreDefinition.java has getOne() that returns Number, while HardSoftScoreDefinition.java has getOneSoftestScore() that returns Score?
I'd prefer them all to just have getOneSoftestScore() that returns Score and avoid any levelNumbers magic.
(LevelNumbers are error prone.)